subhub/api/payments.py

```diff
@@ -158,7 +167,9 @@ def delete_customer(uid) -> FlaskResponse:
         return dict(message="Customer does not exist."), 404
     deleted_payment_customer = stripe.Customer.delete(subscription_user.cust_id)
     if deleted_payment_customer:
-        deleted_customer = g.subhub_account.remove_from_db(uid)
+        deleted_customer = g.subhub_account.mark_deleted(uid)
+        my_user = g.subhub_account.get_user(uid)
+        logger.info("deleted customer", customer=my_user.customer_status)
```

Comment:
why not `customer_status=user.customer_status`?

Reply:
valid, fixed.